booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,354 @@
+              "NetValue": 18201.97,
+              "Breakdown": {
+                "Items": [
+                  {
+                    "TaxRateCode": null,
+                    "NetValue": 18.11,
+                    "TaxValue": 0.0
+                  },
+                  {
+                    "TaxRateCode": "CZ-L",
+                    "NetValue": 18181.82,
+                    "TaxValue": 1818.18
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 2.04,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            },
+            "AverageAmountPerTimeUnit": {
+              "Currency": "EUR",
+              "GrossValue": 10010.08,
+              "NetValue": 9100.99,
+              "Breakdown": {
+                "Items": [
+                  {
+                    "TaxRateCode": null,
+                    "NetValue": 9.06,
+                    "TaxValue": 0.0
+                  },
+                  {
+                    "TaxRateCode": "CZ-L",
+                    "NetValue": 9090.91,
+                    "TaxValue": 909.09
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 1.02,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RateGroups` | array of [Rate group](hotels.md#rate-group) | required | Information about all available rate groups. |
+| `Rates` | array of [Rate](hotels.md#rate) | required | Information about all available rates. |
+| `CategoryPrices` | array of [Category price](#category-price) | required | Prices for all specified categories. |
+
+#### Category price 
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of category. |
+| `OccupancyPrices` | array of objects with arrays of [Occupancy](#occupancy) | required | List of occupancies for age categories against which rate group prices are supplied. |
+| `RateGroupPrices` | array of [Rate group price](#rate-group-price) | required | Prices for the given category for each of the occupancy bands specified in OccupancyPrices. |
+
+#### Occupancy
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AgeCategoryId` | string | required | Identifier of age category. |
+| `PersonCount` | number | required | Number of persons of this Age category type. |
+
+#### Rate group price
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `MinRateId` | string | required | Unique identifier of rate. |
+| `MinPrice` | [Amount](#amount) objects | required | TotalAmount contains total price for the specified interval for rate, AverageAmountPerTimeUnit contains price for one timeUnit. (Day/Night) |
```

Comment:
The field `MinPrice` should be of type object [Min price]; then `Min price` should be defined underneath, consisting of two fields, `TotalAmount` and `AverageAmountPerTimeUnit`, those are both of type `Amount`; and the same for `MaxPrice`.